MonstieWash/Assets/Scripts/Player/PlayerHealth.cs

```diff
@@ -163,5 +185,52 @@ IEnumerator StartInvincibility()
         }
 
         m_isInvincible = false;
-    }    
+    }
+
+    /// <summary>
+    /// Applies a slow and knockback away from the centre of the monster, knockback and slow reduce over time.
+    /// </summary>
+    /// <returns></returns>
+    IEnumerator ApplyKnockbackEffects()
+    {
+        //Initialise time tracking and set hand move speed to starting slow strength.
+        var timeRunning = 0f;
+        m_hand.HandSpeed = Mathf.Max(m_originalMoveSpeed - speedCurve.Evaluate(0 / slowDuration) * slowStrength, 0);
+
+        //Get Hands location
+        var handToV2 = new Vector2(m_hand.transform.position.x, m_hand.transform.position.y);
+        //Get the centre of the screen.
+        Vector2 centre;
+        if (knockBackCentralPoint != null)
+        { centre = new Vector2(knockBackCentralPoint.position.x, knockBackCentralPoint.position.y) - handToV2; }
+        else
+        { centre = Camera.main.ScreenToWorldPoint(new Vector3(Screen.width * 0.5f, Screen.height * 0.5f, 0)); };
+
+        //Calculate knockback from point provided.
+        var dir = new Vector2(handToV2.x - centre.x,handToV2.y - centre.y);
+
+        //If slow or knockback hasn't completed continue running.
+        while (timeRunning < knockbackDuration || timeRunning < slowDuration)
+        {
+            //Move hand in direction of knockback by animation curve strength at time.
+            if (timeRunning < knockbackDuration)
+            { 
+            var moveVec = dir.normalized * knockbackCurve.Evaluate(timeRunning / knockbackDuration) * knockbackInitialStrength * Time.deltaTime;
+            m_hand.gameObject.transform.position += new Vector3(moveVec.x, moveVec.y, 0);
+            }
+
+            //Update slow strength by animation curve over time.
+            if (timeRunning < slowDuration)
+            {
+                m_hand.HandSpeed = Mathf.Max( m_originalMoveSpeed - speedCurve.Evaluate(timeRunning / slowDuration) * slowStrength, 0);
+            }
+
+            timeRunning += Time.deltaTime;
+            print(timeRunning);
```

Comment:
might be a good idea to add a serialized `debug` bool like the other scripts to toggle this on and off.